vm/src/version.rs

```diff
@@ -7,7 +7,7 @@ const MICRO: usize = 0;
 const RELEASELEVEL: &str = "alpha";
 const SERIAL: usize = 0;
 
-#[pystruct_sequence(name = "version_info")]
+#[pystruct_sequence(name = "sys.version_info")]
```

Comment:
The name is `version_info` and `sys` is module name.
```
>>> import sys
>>> type(sys.version_info).__name__
'version_info'
>>> type(sys.version_info).__module__
'sys'
```

Reply:
Thank you for advice. I'll fix it asap.
Yesterday, I found similar struct which has same problem.
https://github.com/RustPython/RustPython/blob/master/vm/src/pyhash.rs#L107-L118
Do I need create a new issue or only PR?